index.html

```diff
@@ -39,5 +36,190 @@ <h2 class="social order">LET'S CONNECT</h2>
+              <li class="list3">2015</li>
+            </ul>
+          </div>
+          <p class="section-description0">A daily selection of privately personalized reads; no
+            accounts or sign-ups
+            required.</p>
+          <ul class="tags">
+            <li class="li11">
+              <article class="text1">html</article>
+            </li>
+            <li class="li12">
+              <article class="text2">css</article>
+            </li>
+            <li class="li13">
+              <article class="text3">javascript
+            </li>
+          </ul>
+          <div class="action">
+            <button class="btn">See Project</button>
+          </div>
+        </div>
+      </div>
+    
+      <div class="section4">
+        <div class="right-block">
+          <img class="image-section4" src="./images/Snap4.png" alt="project image">
+        </div>
+        <div class="left-block">
+          <div class="experience1">
+            <h1 class="section-title1">Multi-Post
+              Stories</h1>
+            <ul class="list">
+              <li class="list1">CANOPY</li>
+              <li class="list2">Back End Dev</li>
+              <li class="list3">2015</li>
+            </ul>
+          </div>
+          <p class="section-description0">A daily selection of privately personalized reads; no
+            accounts or sign-ups
+            required.</p>
+          <ul class="tags">
+            <li class="li11">
+              <article class="text1">html</article>
+            </li>
+            <li class="li12">
+              <article class="text2">css</article>
+            </li>
+            <li class="li13">
+              <article class="text3">javascript
+            </li>
+          </ul>
+          <div class="action">
+            <button class="btn">See Project</button>
+          </div>
+        </div>
+      </div>
+    
+      <div class="section5">
+        <div class="top">
+          <h4 class="about-heading">About <br>Myself</h4>
+          <p class="about-description">Hello I'm a software developer! I can help you build a product, feature or website. Look
+            through some of my work and experience. If you like what you see and have a project you need coded, don't
+            hestiate to contact me.</p>
+          <h2 class="social order">LET'S CONNECT</h2>
+
+          <ul class="social-links">
+            <li><a href="https://twitter.com/YoungeCjay" target="_blank"><i class="fa-brands fa-twitter li1"></i></a></li>
+            <li><a href="https://www.linkedin.com/in/clinton-younge-83386a25a/" target="_blank"><i class="fa-brands fa-linkedin li2"></i></a></li>
+            <li><a href="#"><i class="fa-brands fa-m li3"></i></a></li>
+            <li><a href="https://github.com/ClintonYounge" target="_blank"><i class="fa-brands fa-github li4"></i></a></li>
+            <li><a href="#"><i class="fa-brands fa-angellist li5"></i></a></li>
+          </ul>
```

Comment:
- [ ] [ OPTIONAL ] Good job applying the font awesome icons, but kindly consider downloading the icons from the Figma design and using them in your project. This will help in having same-sized icons and avoid mixing up colors in the icons.